developerportal/apps/videos/models.py

```diff
@@ -263,7 +263,7 @@ def read_time(self):
     def related_resources(self):
         """Returns resources that are related to the current resource, i.e. live,
         public articles and videos which have the same topics."""
-        topic_pks = self.topics.values_list("topic")
+        topic_pks = [topic.topic.pk for topic in self.topics.all()]
```

Comment:
I can see how this caused the blow-up, in that the old version returned instances, not ids/pks.

Would be good to add/improve test coverage for all custom code, but as a fix for now, I trust @liamnewmarch that this is legit

Reply:
> the old version returned instances, not ids/pks

Yes, it’s interesting that it worked at all.